### PR TITLE
Adding Aditya's 2023 Symposium Info

### PR DESCRIPTION
**Pull request by aditya-uw**
I'll also consider updating any personal page info with this PR but wanted to start off with addressing the main things!

### CODE REVIEW

**Comment by aditya-uw** [comment]
@leewujung there are some jobs that are hung currently. However, I think I'm done? Feel free to suggest anything you find!

File: content/newslist.dat [hunk 1]
```diff
@@ -1,3 +1,5 @@
+**[05/19/2023]** Aditya presented on his ongoing research into subsampling for passive acoustic monitoring of bats at the [UW's 26th Annual Undergraduate Research Symposium](https://expo.uw.edu/expo/apply/676/proceedings).
```
Comment (leewujung) [inline content/newslist.dat:1]:
```suggestion
**[05/19/2023]** Aditya presented his ongoing research on the effects of subsampling for passive acoustic monitoring of bats at [UW's 26th Annual Undergraduate Research Symposium](https://expo.uw.edu/expo/apply/676/proceedings).

```

**Review by leewujung — APPROVED** [review]
@aditya-uw : looks great! I made a minor suggestion. Feel free to merge afterwards!

**Comment by aditya-uw** [comment]
Oh @leewujung, "only those with write access to this repository can merge pull requests", implying that I cannot merge. I'm good with your suggestion though so feel free to merge it for me!

**Comment by aditya-uw** [comment]
@leewujung the merge ended up hanging too :(

**Comment by leewujung** [comment]
We're all good!

**Pull request merged**